modules/nodes-pods-autoscaling-creating-web-console.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-creating-web-console_{context}"]
+
+= Creating a horizontal pod autoscaler using web console
+
+You can create a horizontal pod autoscaler to specify the minimum and maximum number of pods you want to run on a standalone deployment or deployment configurations. You can define the CPU utilization or memory utilization your pods should target.
+
+[NOTE]
+====
+Horizontal pod autoscaler cannot be added to those Deployment or Deployment Configs that are part of Operator Backed Service, Knative Service, or Helm Chart.
+====
+
+.Procedure
+
+You can create, edit, and remove a horizontal pod autoscaler in the web console.
+
+* Create a horizontal pod autoscaler:
+** In the *Topology* view, click the node to reveal the side pane.
+** Click *Actions* drop down and select the *Add HorizontalPodAutoscaler*, that will open the *Add HorizontalPodAutoscaler* form page.
+.Add HorizontalPodAutoscaler
+image::node-add-hpa-action.png[]
+** In the form page define name, minimum and maximum pod limits, CPU utilization, and memory utilization, then click *Save*.
++
+[NOTE]
+====
+If any of the values for CPU and memory utilization are missing you will be notified by a warning.
```

Comment:
If any of the values for CPU and memory use are missing, a warning is displayed.